### PR TITLE
[PUB-1910] Add note view for story group composer

### PR DESCRIPTION
**Pull request by albennett**
## Description
Add note view for story group composer
https://buffer.atlassian.net/browse/PUB-1910
http://hi.buffer.com/b9a5ea3f4a85

## Context & Notes
- Karel, I copied over your HeaderBar component from the Add Story PR. 
- Julianna is currently working on the textarea BDS component. I temp copy pasted text area component from hashtag manager but will change it with BDS once finished.
- There might need to be more changes made to this component once we start piecing everything together.

## Checklist

-   [x] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [x] I have tested different user stories. 
-   [x] I have identified similar or related features and tested they are still working.
-   [x] I have performed a self-review of my own code.
-   [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [x] I have commented my code, particularly in hard-to-understand areas.

#### Staging Deployment

To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`

### CODE REVIEW

File: packages/story-group-composer/components/AddNote/index.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+const BodyWrapperStyle = styled.div`
+  display: flex;
+`;
+
+const ButtonStyle = styled.div`
+  margin-left: auto;
+  margin-right: 5px;
+`;
+
+const imageStyle = {
```
Comment (karelm) [inline packages/story-group-composer/components/AddNote/index.jsx]: can you turn this into a StyledImage = styled.img`...`?

Comment (albennett) [reply packages/story-group-composer/components/AddNote/index.jsx]: Great question! I tried this, but couldn't figure out how to add the source attribute this way unless we used background-image 🤔

Comment (karelm) [reply packages/story-group-composer/components/AddNote/index.jsx]: can you not just use ``StyledImage = styled.img`...`; <StyledImage src={story.thumbnail} />`` ?

Comment (albennett) [reply packages/story-group-composer/components/AddNote/index.jsx]: ha yes, that's a great point. thanks!

**Pull request merged**